src/Controllers/TokenController.cs

```diff
@@ -108,7 +108,7 @@ public async Task<ActionResult> GetACSTokenAsync()
         /// <response code="401">Fail to get an authorization code from the request header.</response>
         /// <returns>If authorizing successfully, an ACS access token for the Teams user. Otherwise, an unauthorized error message.</returns>
         [HttpGet]
-        [Route("aad")]
+        [Route("teams")]
```

Comment:
@lei-msft  Should the endpoint documentation be also added in endpoint-and-responses file.

Reply:
Already done in previous pr